app/code/Magento/Ui/view/base/web/js/form/element/abstract.js

```diff
@@ -296,7 +296,7 @@ define([
                 this.validation[rule] = options;
             }
 
-            changed = utils.compare(rules, this.validation).equal;
+            changed = !(utils.compare(rules, this.validation).equal);
```

Comment:
Hi @kisroman 
Please see the Travis report - remove the unnecessary parentheses.

Reply:
Done.